app/com/arpnetworking/rollups/RollupGenerator.java

```diff
@@ -483,13 +500,22 @@ private MetricsQuery buildLastDataPointQuery(
                 : lastCompletePeriod;
     }
 
+    private Optional<String> getNamespace(final String metricName) {
```

Comment:
We can't hard-code this here.

We should support one or both of:
* Tag promotion; configurable tag names to promote from context to self-instrumentation
* Metric name prefix; configurable metric name prefix 

These were both implemented in KDB. And the former in MAD and CAGG as well.

Reply:
I just went off your recent KDB implementation, supporting only metric name (prefix) for now.